app/_locales/en/messages.json

```diff
@@ -2250,6 +2250,16 @@
   "nftTokenIdPlaceholder": {
     "message": "Enter the token id"
   },
+  "nftWarningContent": {
+    "message": "You're granting access to $1, including any you might own in the future. The party on the other end can transfer these NFTs from your wallet at any time without asking you until you revoke this approval. $2"
```

Comment:
I'm not sure what the best practices for content are but should there be a description explaining what `$1` and `$2` are similarly to `nftWarningContentBold`?

Reply:
done